contracts/RMRK/nestable/IRMRKNestable.sol

```diff
@@ -185,7 +191,8 @@ interface IRMRKNestable is IERC165 {
     /**
      * @notice Used to reject all pending children of a given parent token.
      * @dev Removes the children from the pending array mapping.
-     * @dev The children's ownership structures are not updated.
+     * @dev This does not update the ownership storage data on children. If necessary, ownership can be reclaimed by the
+     *  rootOwner of the previous parent.
```

Comment:
I'm not sure this is worth adding here.
The thing is, it does not updated it because from the child's perspective this guy is already the parent.

Reply:
I'd leave it. It might be a bit redundant, but might help someone when reading through the docs 🤔